docs/faq.md

```diff
@@ -108,6 +110,16 @@ network:
       dhcp-identifier: mac
 ```
 
+## Resolving the VMs IP with bridged networking
+
+When running `tart run` with `--net-bridged`, you need to invoke `tart ip` differently, because the macOS built-in DHCP server won't have any information about the VM's IP-address:
+
+```shell
+tart ip --resolver=arp <VM>
+```
+
+This causes the `tart ip` to consult the host's ARP table instead of the `/var/db/dhcpd_leases` file.
```

Comment:
I have very little experience with querying arp, but in the last time I tried to use it the vm (not a tart vm) got an ip and was accessible, but arp did not report the vm ip.

Reply:
Probably related to https://github.com/cirruslabs/tart/issues/993#issuecomment-2586640804.

I'll try to document this in this section too.

Reply:
This seems to the common case, see
https://github.com/cirruslabs/tart/issues/873#issuecomment-2602347271

Reply:
Improved in https://github.com/cirruslabs/tart/pull/1009/commits/c21cebfb2d029e6ab45b244e82b87d3935d797b2.